engine/access/rpc/backend/accounts/handler/handler.go

```diff
@@ -0,0 +1,16 @@
+package handler
```

Comment:
let's avoid the term "handler" for these, since that's already used by the API for the protocol specific logic (grpc/rest/etc)
https://github.com/onflow/flow-go/blob/master/engine/access/rpc/handler.go#L28